axelor-cash-management/src/main/resources/i18n/messages_fr.csv

```diff
@@ -30,11 +30,11 @@
 "<h4>Forecasts in out total</h4>","<h4>Prévisionnel de trésorerie</h4>",,
 "Amount","Montant",,
 "App",,,
-"App cash management",,,
+"App cash management","Prévisions de trésorerie",,
 "Balance",,,
 "Bank","Banque",,
 "Bank details",,,
-"Base",,,
+"Base","Montant de base",,

```

Comment:
Not sure about this

Reply:
Cash management forecast : Opportunity amount instead of Best / Worse case